src/docfx/lib/log/Telemetry.cs

```diff
@@ -69,6 +69,11 @@ public static void TrackBuildCommitCount(int count)
             s_buildItemCountMetric.TrackValue(count, TelemetryName.BuildCommits.ToString(), s_os, s_version, s_repo, s_branch);
         }
 
+        public static void TrackEvent(string name, Dictionary<string, string> properties, Dictionary<string, double> metrics = null)
```

Comment:
nit: consider a more descriptive method like `TrackConfig(config)`

Reply:
so we prefer to hide telemetry specific logic in `Telemetry`? I think that's a good idea.